examples/cpp/parabolic1D.cpp

```diff
@@ -0,0 +1,40 @@
+#include <iostream>
+#include <math.h> 
+#include "mole.h"
+/**
+ * This example uses MOLE to solve the heat equation u_t-u_xx=0,u(x,0)=100x^2,u(-1,t)=100,u(1,t)=100
```

Comment:
In the spirit of matching the MATLAB version, it should solve u_t - alpha*u_xx = 0. And remove the "true solution" part, "true solution" and "approximately" don't go well together.

Reply:
Hi @aneeshs1729 , please address Jared comments.